packages/stokhos/src/CMakeLists.txt

```diff
@@ -804,6 +807,15 @@ IF (Stokhos_ENABLE_Sacado)
           "${STOKHOS_DEPLIBS}"
           DEPLIBS_OUT)
         LIST(APPEND STOKHOS_TPETRA_DEPLIBS ${DEPLIBS_OUT})
+       PROCESS_ENSEMBLE_ETI(
+          "${Stokhos_ETI_Ensemble_Sizes}"
+          "${TPETRA_SD_ETI_CLASSES}"
```

Comment:
This is much better -- I was thinking on the way home about setting up a table of ETI classes, that lists a subset of `{S, LO, GO, N}` over which to instantiate each.